app/com/openseedbox/jobs/NodePollerJob.java

```diff
@@ -38,7 +43,7 @@ protected Object doGenericJob() throws Exception {
 					db.merge(match);
 				}
 			}
-			Torrent.batch().update(fromDb);
+			Torrent.save(fromDb);
```

Comment:
Simply roll back the unrelated change! 
```suggestion
			Torrent.batch().update(fromDb);
```